napari/components/_viewer_key_bindings.py

```diff
@@ -155,3 +155,12 @@ def hold_for_pan_zoom(viewer: ViewerModel):
         yield
 
         selected_layer.mode = previous_mode
+
+
+@register_viewer_action(trans._("Show all key bindings"))
```

Comment:
again @lucyleeow remind me why we moved this sorry? It was just in the `file_menu` file?

Reply:
Yes it was in `file_menu.py` and needed to be moved since we deleted that file and we decided this would be the best place for it.